roles/lib/files/FWO.Report.Filter/DynGraphqlQuery.cs

```diff
@@ -242,8 +244,7 @@ private static void SetFixedFilters(ref DynGraphqlQuery query, ReportTemplate re
         {
             // leave out all header texts
             if (reportParams.ReportParams.ReportType != null &&
-                reportParams.ReportParams.ReportType == (int)ReportType.Statistics &&
-                reportParams.ReportParams.ReportType != (int)ReportType.Recertification)
+                reportParams.ReportParams.ReportType == (int)ReportType.Statistics)
```

Comment:
if ReportType is Statistics, it can't be Recertification anyway. So I guess it was meant: 
if (reportParams.ReportParams.ReportType != null &&
                (reportParams.ReportParams.ReportType == (int)ReportType.Statistics ||
                reportParams.ReportParams.ReportType == (int)ReportType.Recertification))